lib/experian/precise_id/client.rb

```diff
@@ -18,9 +18,23 @@ def send_answers(options = {})
 
       def submit_request(request)
         raw_response = super
-        log_raw_response(raw_response)
         response = Response.new(raw_response.body)
+        check_response_for_errors(response, raw_response)
         [request,response]
+      rescue => e
+        log_error(e)
+        log_raw_response(raw_response)
+      end
+
+      def check_response_for_errors(response, raw_response)
+        if Experian.logger && response.error? && (response.error_code.nil? && response.error_message.nil?)
```

Comment:
I wrote this code many moons ago... and looking into it I see [we check for errors here](https://git.sittercity.com/sittercity/member_verification_api/blob/master/lib/member_verification/verify.rb#L29-L33). I think that maybe that's a better place for this code... I'll see if there is a way to get the raw response out of the returned result and we can delete this code.